src/playkit.js

```diff
@@ -33,6 +33,8 @@ import {AdBreakType} from './ads/ad-break-type';
 import {AdTagType} from './ads/ad-tag-type';
 import {AdEventType} from './ads/ad-event-type';
 import {ScreenOrientationType} from './screen-orientation-type';
+import 'intersection-observer';
```

Comment:
why it's needed?

Reply:
ie11 support. moved to viewabilityManager